packages/core/styles/components/_dialog.scss

```
@@ -35,20 +36,25 @@ dialog {
+    width: 100%;
 
-    & + p {
+    &+p {
       margin-top: var(--ix-space-sm);
     }
   }
 
   [data-actions] {
     display: flex;
     align-items: center;
+    justify-content: flex-end;
```

Stylelint (17.12.0)

[error] Duplicate property "justify-content" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)